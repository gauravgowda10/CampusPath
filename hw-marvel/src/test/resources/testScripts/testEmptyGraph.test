# Test empty graph

# Load graph
LoadGraph empty empty.tsv

# List nodes
ListNodes empty

# Find path
FindPath empty node1 node2