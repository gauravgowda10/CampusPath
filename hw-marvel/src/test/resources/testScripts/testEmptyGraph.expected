# Test empty graph

# Load graph
loaded graph empty

# List nodes
empty contains:

# Find path
unknown character node1
unknown character node2